package.json

```diff
@@ -7,6 +7,7 @@
   },
   "author": "Vinay Pulim <v@pulim.com>",
   "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
Can you change it to `>=0.21.1`?

Reply:
@RopoMen ping

Reply:
@orgads it is now fixed.

Reply:
Please mark this conversation as resolved.